CHANGELOG.md

```diff
@@ -1,19 +1,51 @@
-
 # Change Log
+
 All notable changes to this project will be documented in this file.
- 
-The format is based on [Keep a Changelog](http://keepachangelog.com/)
-and this project adheres to [Semantic Versioning](http://semver.org/).
- 
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to
+[Semantic Versioning](http://semver.org/).
+
+## [1.1.0] 2024-06-28
```

Comment:
Not sure how this date is created, but should it be updated to today's date?

Reply:
Yes, I think that is better to update the date. I will create another PR with the updated date.